### ISSUE TITLE
Should we have a dataset of tech companies what data privacy / protections they offer?

### ISSUE THREAD

**Issue by bnvk** [digitalfreedom/discussion#1]
For example, a dataset that records stuff like:

![apple-data-icloud-data-privacy](https://cloud.githubusercontent.com/assets/48677/9744310/6806dbc6-566b-11e5-9b74-8d0cd59deb35.png)

From [source articles](http://www.nytimes.com/2015/09/08/us/politics/apple-and-other-tech-companies-tangle-with-us-over-access-to-data.html) or is that starting to cast our net a little bit too wide?

**Comment by seamustuohy** [comment]
Might check out [Ranking Digital Rights Corporate Accountability Index](https://rankingdigitalrights.org/) to see if you can pull some info directly from there.

**Comment by bnvk** [comment]
@elationfoundation cool stuff, thanks. I just opened another broader thread #2 about other potential sets. Do you know the people at the org you posted above enough to introduce / connect us?

**Comment by seamustuohy** [comment]
@bnvk Yeah, I will shoot an e-mail introducing you. Also, let's jump on a call in the next couple of weeks (I'm headed to Berlin next week and will be sequestered into the device-less Gunner show most of the time.) Your brushing up against something that i've been putting a lot of time into recently and it seems like the environment is ripe for integrating the efforts of a few communities I have been talking to.

**Comment by bnvk** [comment]
@elationfoundation wunderbar! I'll email you my various alt comm channels- would be great to collaborate a bit :-)
